src/pages/career.tsx

```diff
@@ -6,16 +6,31 @@ import { CareerPage } from '../components/pages/career/career-page';
 import { SyPersonioJob } from '../types';
 import { IGatsbyImageData } from 'gatsby-plugin-image';
 
+interface OfficeImage {
+  relativePath: string;
+  childImageSharp: {
+    gatsbyImageData: IGatsbyImageData;
+  };
+}
+
 interface CareerPageQueryProps {
   allSyPersonioJob: {
     nodes: SyPersonioJob[];
   };
+  officeImages: {
+    nodes: OfficeImage[];
+  };
   hero: IGatsbyImageData;
 }
 
 const Career = (props: PageProps<CareerPageQueryProps>) => {
   const { t } = useTranslation();
 
+  const officeImages = props.data.officeImages.nodes.reduce((memo, image) => {
+    memo[image.relativePath] = image;
+    return memo;
+  }, {});
+
```

Comment:
Nice idea 🙌 This makes it very easy to access the images in the `CareerPage` component